x-pack/plugins/triggers_actions_ui/public/application/components/simple_connector_form.tsx

```diff
@@ -32,35 +39,45 @@ interface SimpleConnectorFormProps {
   readOnly: boolean;
   configFormSchema: ConfigFieldSchema[];
   secretsFormSchema: SecretsFieldSchema[];
+  configFormSchemaAfterSecrets?: ConfigFieldSchema[];
```

Comment:
Another solution would be to extend `ConfigFieldSchema` with a `afterSecrets: boolean` attribute and do the partition in the `SimpleConnectorFormComponent`. This way consumers will not have to create two configurations.

Reply:
You might need both, I do not know!